lib/tasks/export.rake

```diff
@@ -76,6 +77,34 @@ def export_question(csv, node, level = nil)
   end
 end
 
+def export_email(email, prefix, zos)
+  subject = (email.subject || 'no subject').gsub(' ', '_').gsub(/[^0-9a-z_]/i, '')
+  # some subjects have special cahrs in them, which messes with the wkhtmltopdf bash
+  # truncate at 200 char or filename migh tbe too long
+  filename = [email.sent_at.to_formatted_s(:number), subject].join('_')[0..200]
+  mk_zip_entry(zos, "#{prefix}/email/#{filename}.eml", email.sent_at) do
+    zos << email.raw_source
+  end
+
+  # pdf creation
+  begin
+    temp_eml = Tempfile.new("#{filename}.eml")
```

Comment:
if we don't use the `filename` in the tempfile name we could skip the filename cleanup on line 81 above, no?

Reply:
I think it would still be an issue with the pdf/em output into the zip, right?

Reply:
it could be.... in that case what do you think about moving the filename cleanup/truncation into the `mk_zip_entry` function?

Reply:
all of it? `.gsub(' ', '_').gsub(/[^0-9a-z_]/i, '')[0..200]`

Reply:
yeah, what do you think?

Reply:
I tried, but it involved parsing the whole path, extension and it barfed every where. Gonna leave as is, unless you want me to take another crack at it.